tox/config.py

```diff
@@ -421,15 +421,9 @@ def basepython_default(testenv_config, value):
         name="envlogdir", type="path", default="{envdir}/log",
         help="venv log directory")
 
-    def downloadcache(testenv_config, value):
-        if value:
-            # env var, if present, takes precedence
-            downloadcache = os.environ.get("PIP_DOWNLOAD_CACHE", value)
-            return py.path.local(downloadcache)
-
     parser.add_testenv_attribute(
-        name="downloadcache", type="string", default=None, postprocess=downloadcache,
-        help="(deprecated) set PIP_DOWNLOAD_CACHE.")
+        name="downloadcache", type="string", default=None,
+        help="(ignored) has no effect anymore, pip-8 uses local caching by default")
```

Comment:
yes, i guess with tox-3.0 we should go through all deprecated and ignored things and finally remove them.  The bulk of the code is already removed, it's really this help string/declaration you commented on that remains wrt downloadcache.